_source/_posts/2019-05-31-simple-auth-express-fifteen-minutes.md

```diff
@@ -44,22 +45,23 @@ Edit the `"scripts"` section of your `package.json` to look like this:
 Now create a new file `index.js`:
 
 **index.js**
+
 ```javascript
-const express = require('express');
-const path = require('path');
+const express = require("express");
+const path = require("path");
```

Comment:
Is there any particular reason you changed all single quotes to double quotes? I believe we tend to use single-quotes in most JavaScript posts. If there's no reason for changing them, we should probably change them back.

Reply:
I formatted markdown using prettier, since its the default, it was changed, I can change back to single quotes, no issues